### PR TITLE
Sequence diagrams v2

### PR DESCRIPTION
**Pull request by Kafcis**
Need to rename a few boxes, can merge first if you want

### CODE REVIEW

File: docs/sequenceDiagrams/Parser.puml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+@startuml
+title Parser
+hide footbox
+
```
Comment (daniellimws) [inline docs/sequenceDiagrams/Parser.puml]: Missing `participant ":Parser" as Parser`

File: docs/sequenceDiagrams/ViewCommand.puml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+@startuml
+hide footbox
+title View
+
```
Comment (daniellimws) [inline docs/sequenceDiagrams/ViewCommand.puml:4]: Missing 

```
participant "command:ViewCommand" as ViewCommand
participant ":Task" as Task
participant "taskList:TaskList" as Task
participant ":Time" as Time
participant ":AthenaUI" as AthenaUI
```

File: docs/sequenceDiagrams/loadStorage.puml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+@startuml
+title LOAD
+hide footbox
+
```
Comment (daniellimws) [inline docs/sequenceDiagrams/loadStorage.puml:4]:
```
participant "storage:Storage" as Storage
participant ":TaskList" as TaskList
participant ":File" as File
```

File: docs/sequenceDiagrams/saveStorage.puml [hunk 4]
```diff
@@ -0,0 +1,35 @@
+@startuml
+title Save
+
```
Comment (daniellimws) [inline docs/sequenceDiagrams/saveStorage.puml:9]:
```
participant "storage:Storage" as Storage
participant ":Time" as Time
participant ":Task" as Task
participant ":TaskList" as TaskList
participant ":File" as File
```

**Comment by daniellimws** [comment]
Oh just realized that you said need to rename a few boxes

**Comment by Kafcis** [comment]
Yeah I wanted to put this here before I passed out

File: docs/sequenceDiagrams/DeleteCommand.puml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+@startuml
+hide footbox
+title DeleteCommand
+participant "command:DeleteCommand" as DeleteCommand
+participant "tasklist:TaskList" as TaskList
+participant "task:Task" as Task
+participant "athenaUi:ATHENAUI" as AthenaUI
```
Comment (amanda-chua) [inline docs/sequenceDiagrams/DeleteCommand.puml]: `AthenaUi` instead of `ATHENAUI`

File: docs/sequenceDiagrams/AddCommand.puml [hunk 6]
```diff
@@ -1,13 +1,13 @@
...
 hide footbox
+title AddCommand
 
-actor "ATHENA" as Athena
-participant ":AddCommand" as AddCommand
-participant ":TaskList" as TaskList
-participant ":Task" as Task
+participant "command:AddCommand" as AddCommand
+participant "tasklist:TaskList" as TaskList
+participant "task:Task" as Task
```
Comment (amanda-chua) [inline docs/sequenceDiagrams/AddCommand.puml:9]: missing AthenaUi

File: docs/sequenceDiagrams/DoneCommand.puml [hunk 7]
```diff
@@ -0,0 +1,28 @@
+@startuml
+hide footbox
+title Done
+
+participant "command:DoneCommand" as DoneCommand
+participant "tasklist:TaskList" as TaskList
+participant "task:Task" as Task
+participant "athenaUi:ATHENAUI" as AthenaUI
```
Comment (amanda-chua) [inline docs/sequenceDiagrams/DoneCommand.puml]: `AthenaUi` instead of `ATHENAUI`

File: docs/sequenceDiagrams/EditCommand.puml [hunk 8]
```diff
@@ -0,0 +1,21 @@
+@startuml
+hide footbox
+title Edit
+
+participant "command:EditCommand" as EditCommand
+participant "tasklist:TaskList" as TaskList
+participant "task:Task" as Task
+
```
Comment (amanda-chua) [inline docs/sequenceDiagrams/EditCommand.puml:9]: missing AthenaUi

**Review by amanda-chua — APPROVED** [review]

**Pull request merged**